src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass_Switch.cs

```diff
@@ -138,7 +138,17 @@ public override BoundNode VisitSwitchDispatch(BoundSwitchDispatch node)
             return null;
         }
 
-        public override BoundNode VisitSwitchExpression(BoundSwitchExpression node)
+        public override BoundNode VisitConvertedSwitchExpression(BoundConvertedSwitchExpression node)
+        {
+            return this.VisitSwitchExpression(node);
+        }
+
+        public override BoundNode VisitUnconvertedSwitchExpression(BoundUnconvertedSwitchExpression node)
```

Comment:
>VisitUnconvertedSwitchExpression [](start = 34, length = 32)

If no unconverted switch expressions survive past initial binding, can we hit this code?

Reply:
No.  And if some do survive, for example in error scenarios, yes.

---
In reply to: [301816043](https://github.com/dotnet/roslyn/pull/37052#discussion_r301816043) [](ancestors = 301816043)